docker/Dockerfile

```diff
@@ -94,6 +94,11 @@ RUN bash ./docker/dist-build.sh
 FROM base as torch-tensorrt
 
 COPY . /opt/torch_tensorrt
+
+# Symlink the path pyenv is using for python with the /opt directory for package sourcing
+RUN mkdir -p "/opt/python3/" &&\
+    ln -s "`pyenv which python | xargs dirname | xargs dirname`/lib/python$PYTHON_VERSION/site-packages" "/opt/python3/"
```

Comment:
Duplicated from above. It is necessary to add this symbolic link again in the main container to ensure rebuilds of Torch-TRT by the user are functional.